src/EditorFeatures/Core/Wrapping/AbstractWrapper.cs

```diff
@@ -21,9 +21,9 @@ namespace Microsoft.CodeAnalysis.Editor.Wrapping
     /// </summary>
     internal abstract partial class AbstractSyntaxWrapper : ISyntaxWrapper
     {
-        protected IBlankLineIndentationService IndentationService { get; }
+        protected Indentation.IIndentationService IndentationService { get; }
```

Comment:
so, making this wrapper using formatter alignment rule didn't work out? we should probably try to do this wrapping stuff using formatting rules to see we can get same result at some point so that we can get rid of 2 code doing same thing.

Reply:
I guess this all indentation stuff happened since people start to add new thing rather than refactor existing one. nice that we at least do that now.